### PR TITLE
Add `twingate_group` resource

### PR DESCRIPTION
**Pull request by sapslaj**
- JIRA Ticket URL: 🙂 i would if i could

## Changes

- Adds `twingate_group` resource.
- Currently can only manage group name

### CODE REVIEW

**Comment by ekampf** [comment]
@sapslaj thanks a lot for contributing!
We're actually in the process of setting ups the infrastructure to support outside contributions - run CI on forked repos, do not require a JIRA ticket.

I have configured CircleCI to run on PRs from forks but I have no way to trigger it for your repo - can you push a change so that it triggers?

**Comment by ekampf** [comment]
@sapslaj awesome!  we'll review and merge ASAP

**Review by ekampf — CHANGES_REQUESTED** [review]
@sapslaj thanks a lot for the PR! We just need a few minor fixes to be able to merge...

**Comment by romankor** [comment]
@sapslaj ping ...

**Comment by sapslaj** [comment]
Hey @romankor and all, haven't forgotten about this just haven't had the time to work on it yet.

**Comment by romankor** [comment]
@sapslaj do you mind if we take over from here ? we have some bandwidth now and can work on the provider

**Comment by vmanilo** [comment]
Added proposed changes in this PR https://github.com/Twingate/terraform-provider-twingate/pull/72

File: docs/resources/group.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the group
+
+### Read-Only
```
Comment (romankor) [inline docs/resources/group.md:22]: Lets add an example of how to use and import groups , like in all other resources 
in order for that to happen you need to add a corresponding file to 
image

**Comment by romankor** [comment]
@sapslaj closing this one , since it was already merge in another PR

**Pull request closed without merging**